great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -116,18 +116,13 @@ def _create(
         runtime_environment: Optional[dict] = None,
     ) -> SerializableDataContext:
         """
-        Build a new great_expectations directory and DataContext object in the provided project_root_dir.
+        Build a new gx directory and DataContext object in the provided project_root_dir.
 
-        `create` will create a new "great_expectations" directory in the provided folder, provided one does not
+        `create` will create a new "gx" directory in the provided folder, provided one does not
         already exist. Then, it will initialize a new DataContext in that folder and write the resulting config.
 
-        --Public API--
-
-        --Documentation--
-            https://docs.greatexpectations.io/docs/terms/data_context
```

Comment:
Just removing this section because:
* it's not in the public API
* the link redirects to 0.18
* Even if the link was up to date, it's going to go out of date at some point.